[2025-06-14T08:32:54.644+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-14T08:32:54.843+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: GlobalSuperStore_dw_etl.extract.extract_and_load_to_staging manual__2025-06-14T08:32:28.782853+00:00 [queued]>
[2025-06-14T08:32:54.857+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: GlobalSuperStore_dw_etl.extract.extract_and_load_to_staging manual__2025-06-14T08:32:28.782853+00:00 [queued]>
[2025-06-14T08:32:54.858+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-06-14T08:32:54.887+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract.extract_and_load_to_staging> on 2025-06-14 08:32:28.782853+00:00
[2025-06-14T08:32:54.898+0000] {standard_task_runner.py:63} INFO - Started process 728 to run task
[2025-06-14T08:32:54.904+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'GlobalSuperStore_dw_etl', 'extract.extract_and_load_to_staging', 'manual__2025-06-14T08:32:28.782853+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/global_superstore_dw_etl.py', '--cfg-path', '/tmp/tmp8w66keo8']
[2025-06-14T08:32:54.910+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask extract.extract_and_load_to_staging
[2025-06-14T08:32:55.178+0000] {task_command.py:426} INFO - Running <TaskInstance: GlobalSuperStore_dw_etl.extract.extract_and_load_to_staging manual__2025-06-14T08:32:28.782853+00:00 [running]> on host 180404561e2e
[2025-06-14T08:32:55.296+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='GlobalSuperStore_dw_etl' AIRFLOW_CTX_TASK_ID='extract.extract_and_load_to_staging' AIRFLOW_CTX_EXECUTION_DATE='2025-06-14T08:32:28.782853+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-14T08:32:28.782853+00:00'
[2025-06-14T08:32:55.297+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-14T08:33:00.087+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-06-14T08:33:00.093+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-14T08:33:00.315+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-14T08:33:00.712+0000] {logging_mixin.py:188} INFO - ✔ Đã load Category → staging.stg_Category
[2025-06-14T08:33:00.719+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-14T08:33:00.727+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-14T08:33:01.539+0000] {logging_mixin.py:188} INFO - ✔ Đã load Customer → staging.stg_Customer
[2025-06-14T08:33:01.571+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-14T08:33:01.577+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-14T08:33:17.199+0000] {logging_mixin.py:188} INFO - ✔ Đã load Order_Details → staging.stg_Order_Details
[2025-06-14T08:33:17.211+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-14T08:33:17.230+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-14T08:33:22.102+0000] {logging_mixin.py:188} INFO - ✔ Đã load Orders → staging.stg_Orders
[2025-06-14T08:33:22.118+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-14T08:33:22.125+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-14T08:33:23.042+0000] {logging_mixin.py:188} INFO - ✔ Đã load Product → staging.stg_Product
[2025-06-14T08:33:23.052+0000] {base.py:84} INFO - Using connection ID 'mysql' for task execution.
[2025-06-14T08:33:23.059+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-14T08:33:23.097+0000] {logging_mixin.py:188} INFO - ✔ Đã load Sub_Category → staging.stg_Sub_Category
[2025-06-14T08:33:23.098+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-06-14T08:33:23.099+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-14T08:33:23.108+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=GlobalSuperStore_dw_etl, task_id=extract.extract_and_load_to_staging, run_id=manual__2025-06-14T08:32:28.782853+00:00, execution_date=20250614T083228, start_date=20250614T083254, end_date=20250614T083323
[2025-06-14T08:33:23.185+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-14T08:33:23.244+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-14T08:33:23.248+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
